portfolio/src/main/webapp/style.css

```diff
@@ -1,12 +1,26 @@
 #content {
-  background-color: white;
-  border: 10px solid red;
-  padding: 100px;
+  text-align: center;
+  padding: 25px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: floralwhite;
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  background:transparent
 }
 
-#fact-container {
-  margin-top: 15px;
+#bold {
+    color: black;
+    text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+}
+
+#button {
```

Comment:
ID names should be as specific as possible, since the represent one particular element on the page. So something like #random-quote-button is a much better ID